pkg/command/service/measure.go

```diff
@@ -626,21 +626,25 @@ kperf service measure --svc-perfix svc --range 1,200 --namespace ns --concurrenc
 				fmt.Printf("Percentile99: %fs\n", measureFinalResult.Result.P99)
 
 				current := time.Now()
-				rawPath := fmt.Sprintf("/tmp/%s_%s", current.Format("20060102150405"), "raw_ksvc_creation_time.csv")
+				outputLocation, err := utils.CheckOutputLocation(measureArgs.output)
+				if err != nil {
+					fmt.Printf("failed to check measure output location: %s\n", err)
+				}
+				rawPath := fmt.Sprintf("%s/%s_%s", outputLocation, current.Format("20060102150405"), "raw_ksvc_creation_time.csv")
```

Comment:
Not directly related to this PR - it's better to define "20060102150405" as a const since I've seen it's used at many places.

Reply:
```suggestion
				rawPath := filepath.Join(outputLocation, fmt.Sprintf("%s_%s", current.Format("20060102150405"), "raw_ksvc_creation_time.csv"))
```
Then you won't need to trim the trailing `/` in `CheckOutputLocation`

Reply:
Thanks, I have changed the code as you suggested.